### DISCUSSION TITLE
Is there any way to get a constant js file name during serving and building?

### DISCUSSION THREAD

**Discussion by rondered** [parcel-bundler/parcel#6496] [Q&A]
Everytime I run serve or build, I get different js file names.
the name option only affecting the html file.

Thanks!

**Comment by aakoch** [comment]
https://v2.parceljs.org/features/production/#content-hashing states:
You can also disable content hashing using the --no-content-hash CLI flag. Note that the name will still include a hash, but it will not change on each build.

I haven't tried it however.
